conda.recipe/meta.yaml

```diff
@@ -0,0 +1,31 @@
+package:
+  name: pystog
+  version: 1.0.0
+
+build:
+  number: 1
+
+source:
+  path: ..
+
+requirements:
+  build:
```

Comment:
We do have more build requirements. One that comes to mind is testing dependencies and linting.

Reply:
Actually, I take this comment back, nevermind. Sorry.